Sources/Sentry/SentryProfiler.mm

```diff
@@ -331,16 +331,17 @@ + (void)startWithTracer:(SentryId *)traceId
         trackProfilerForTracer(_gCurrentProfiler, traceId);
         // record a new metric sample for every concurrent span start
         [_gCurrentProfiler->_metricProfiler recordMetrics];
-        return;
+        return NO;
```

Comment:
`m`: Shouldn't this be `YES`? We are profiling in that case.

```suggestion
        return YES;
```

Reply:
ay, good catch!